src/styles/blocks/stars.scss

```
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 16px;
+
+  &__list {
+    display: flex;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

```suggestion
      &:last-child {
        margin-right: 0;
      }

     &:nth-child(-n + 4) {
       background-image: url(../images/star-active.svg);
     }
  }
```
use parent selector &